.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+# To get started with Dependabot version updates, you'll need to specify which
```

Comment:
Probably we could leave out only two last lines in the template comment, or even get rid of those as well. Not a big deal though.